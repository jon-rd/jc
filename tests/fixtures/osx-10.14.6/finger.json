[{"login":"kbrazil","name":"kbrazil","tty":"con","idle":"15d","login_time":"Mar 22 07:33","tty_writeable":false,"idle_minutes":0,"idle_hours":0,"idle_days":15,"total_idle_minutes":21600},{"login":"kbrazil","name":"kbrazil","tty":"s00","idle":null,"login_time":"Mar 22 07:34","tty_writeable":true,"idle_minutes":0,"idle_hours":0,"idle_days":0,"total_idle_minutes":0},{"login":"kbrazil","name":"kbrazil","tty":"s00","idle":"8","tty_writeable":true,"idle_minutes":8,"idle_hours":0,"idle_days":0,"total_idle_minutes":8},{"login":"kbrazil","name":"kbrazil","tty":"s00","idle":"1:19","login_time":"Mar 22 07:34","tty_writeable":true,"idle_minutes":19,"idle_hours":1,"idle_days":0,"total_idle_minutes":79}]
