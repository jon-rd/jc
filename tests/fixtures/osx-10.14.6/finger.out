Login    Name                 TTY  Idle  Login  Time   Office  Phone
kbrazil  kbrazil             *con   15d  Mar 22 07:33
kbrazil  kbrazil              s00        Mar 22 07:34
kbrazil  kbrazil              s00     8  Tue    09:04
kbrazil  kbrazil              s00  1:19  Mar 22 07:34
